tests/serializer/test_serializer_thrift.py

```diff
@@ -35,11 +31,3 @@ def test_header(v1):
     assert v1 == serializer.deserialize_header(
         serializer.serialize_header(v1)
     )
-
-
-def test_body():
```

Comment:
this test uses generated code and tests functionality that will be deleted in this release, so I axed it